qemu/tests/numa_stress.py

```diff
@@ -35,6 +36,23 @@ def max_mem_map_node(host_numa_node, qemu_pid):
     return (node_map_most, memory_sz_map_most)
 
 
+def get_tmpfs_write_speed():
+    """
+    Get the tmpfs write speed of the host
+    return: The write speed of tmpfs, the unit is kb/s.
+    """
+    utils.run("mount -t tmpfs none /tmp")
```

Comment:
don't use /tmp driectly, some process write temp file in /tmp when avocado running,  remount maybe cause random test failed.
suggest to make a new dir in /tmp folder, then mount tmpfs to it.